connect/res/layout/com_telenor_connect_login_button_with_progress_bar.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    >
+    <com.telenor.connect.ui.ConnectLoginButtonWithoutProgressBar
```

Comment:
The file says *with* progress bar, while this line of code says *without*. It's not immediately apparent to me why they're the opposite.